datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -206,6 +206,56 @@ query error Execution error: In an aggregate with DISTINCT, ORDER BY expressions
+
+query ?? rowsort
+with tbl as (SELECT * FROM (VALUES ('xxx', 'yyy'), ('xxx', 'yyy'), ('xxx2', 'yyy2')) AS t(x, y))
+select
+    array_agg(distinct x order by x) as x_agg,
```

Comment:
These columns already look distinct to me  -- maybe testing with another column would be better to show that distinct is actually working as designed